tests/integration/server/users_api_test.py

```diff
@@ -362,3 +363,39 @@ def test_reject_users_proposals():
         )
         conn.close()
         assert proposal_1_result[0]["status"] == "REJECTED"
+
+
+def test_check_admin_status():
+    """Test that checking a users admin status returns the correct things."""
+    admin_user = {
+        "name": "admin nadia",
+        "username": "admin_nadia",
+        "password": "test11",
+        "email": "admin_nadia@test.com",
+    }
+
+    user = {
+        "name": "nadia four",
+        "username": "nadia4",
+        "password": "test11",
+        "email": "nadia4@test.com",
+    }
+    with requests.Session() as session:
+        admin_response = create_test_user(session, admin_user)
+        non_admin_response = create_test_user(session, user)
+
+        admin_id = admin_response.json()["data"]["user"]["id"]
+        non_admin_id = non_admin_response.json()["data"]["user"]["id"]
+
+        next_admins = {
+            "name": "NextAdmins",
+            "owners": admin_id,
+            "administrators": admin_id,
+            "description": "Next Administrator role",
+        }
+        role_response = create_test_role(session, next_admins)
+
+        add_role_member(session, role_response.json()["data"]["id"], {"id": admin_id})
+
+        assert check_admin_status(admin_id)
```

Comment:
Tests should assert whether the check_admin_status() returns True or False for the admin user and the non admin user.

Reply:
looks good once these changes are made.